README.md

```diff
@@ -0,0 +1,26 @@
+# Lissi Connect through NodeJS
+
+This express application presents a working example of a Single Sign-On method through lissi-Connect using `passportJS` and `openid-client` strategy.
```

Comment:
It is not `single sign-on` but `passwordless authentication` or `passwordless login`